webapp/portlet/src/main/webapp/space-settings/components/SpaceSettings.vue

```diff
@@ -29,6 +29,11 @@ export default {
     document.addEventListener('hideSettingsApps', () => this.displaySpaceExternalSettings = false);
     document.addEventListener('showSettingsApps', () => this.displaySpaceExternalSettings = true);
   },
+  destroyed() {
+    document.removeEventListener('chat-external-updated');
```

Comment:
Is this needed ? we do not want to add any specific event here

Reply:
Not really i will close this PR.